includes/tracker.php

```diff
@@ -1,6 +1,10 @@
 <?php
 namespace Elementor;
 
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
```

Comment:
Can you use the Admin Settings API to get this data?
The goal is that if we will add more settings it will automatically appear here by receiving the settings keys and values directly from the Settings API

Reply:
Where i can see this API?

Reply:
Handled.